README.md

```diff
@@ -13,11 +13,12 @@ attest,
    ```yaml
    permissions:
      id-token: write
-     contents: write
+     attestations: write
+     contents: read # optional, usually required
```

Comment:
I would love to know more about this! https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#permissions says that this would allow you to list the commits in a repository, but there must be something else we're using it for?

Or is it that this is part of the "default permissions" if you don't specify a `permissions:` section, but since we **are** specifying a `permissions:` section we have to include it?

Reply:
i _think_ once you're not using the default permissions `contents` will be set to `none`, so the `actions/checkout` call lower down will fail.